server/mergin/sync/private_api.yaml

```diff
@@ -26,13 +26,21 @@ paths:
           required: true
           schema:
             type: string
+        - $ref: "#/components/parameters/Page"
```

Comment:
is there any reason pagination params are not merged into single reference?

Reply:
parameter are not possible to merge to one type: object ref, because there have to be name parameter in PaginatedParams:

```
parameters:
    PaginatedParams:
      in: query
      required: true
      **name: pagination**
      schema:
        type: object
        properties:
          page:
            type: integer
            minimum: 1
            example: 1
          per_page:
            type: integer
            maximum: 100
            example: 50
```

Requests will be instead ?page=1&per_page=1 -> ?pagination.page=1&pagination.per_page=1